src/test_typing_extensions.py

```diff
@@ -160,6 +160,32 @@ def test_exception(self):
             assert_never(None)
 
 
+class AnyTests(BaseTestCase):
+    class SubclassesAny(Any):
```

Comment:
Let's put this into an actual test case?
Wouldn't hurt to copy over the tests from `test_can_subclass` in the original PR; I seem to recall there was some trickiness wrt inheritance, so might as well test it.